anchore_engine/db/entities/common.py

```diff
@@ -126,7 +126,9 @@ def do_connect(db_params):
 
     db_connect = db_params.get('db_connect', None)
     db_connect_args = db_params.get('db_connect_args', None)
-    db_engine_args = db_params.get('db_engine_args', {})
+    db_engine_args = db_params.get('db_engine_args')
```

Comment:
I think this change doesn't introduce any different functionality?

Reply:
This was to handle the case where there is a 'null' value in the yaml config directly. Ensures that we get an empty dict. Lines 130 & 131 unify the cases. May be able to be handled more elegantly.